package/src/skia/web/JsiSkPaint.ts

```diff
@@ -66,6 +66,10 @@ export class JsiSkPaint extends HostObject<Paint, "Paint"> implements SkPaint {
     this.ref.setAntiAlias(aa);
   }
 
+  setDither(aa: boolean) {
+    this.ref.setDither(aa);
```

Comment:
setDither is not exposed in canvas kit, could that be?

Reply:
Yes, see comment below. We would need to expose it...